src/AspNet.Security.OAuth.PingOne/AspNet.Security.OAuth.PingOne.csproj

```
@@ -0,0 +1,24 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>$(DefaultNetCoreTargetFramework)</TargetFrameworks>
+  </PropertyGroup>
+
+
+  <!-- TODO Remove once published to NuGet.org -->
+  <PropertyGroup>
+    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

As this package doesn't exist yet, it needs to specify the version where it _will_ exist so that builds post-merge work correctly where package validation is enforced.

```suggestion
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
    <PackageValidationBaselineVersion>7.0.1</PackageValidationBaselineVersion>
```